view/src/__tests__/Callstack.test.jsx

```diff
@@ -0,0 +1,129 @@
+// Unit tests of Callstack component
+// Author: Lukas Petr
+
+import {
+  fireEvent, render, screen,
+} from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import Callstack from '../components/Callstack';
+
+describe('simple callstack visualisation test', () => {
+  const compFunName = '__put_task_struct';
+  const oldCallStack = [{ name: 'free_task' }, { name: 'free_task_struct' }];
+  const newCallStack = [{ name: 'free_task' }, { name: 'free_task_struct' }];
+  const onSelect = jest.fn();
+
+  const setup = () => {
+    render(
+      <Callstack
+        compFunName={compFunName}
+        oldCallStack={oldCallStack}
+        newCallStack={newCallStack}
+        definitions={{}}
+        onSelect={onSelect}
+      />,
+    );
+  };
+  it('all function names should be visible', () => {
```

Comment:
General comment: looking at your test descriptions, I think that it would be more appropriate to use `test` instead of `it`. AFAIU, `it` should be used in the form:
```
it("should do something")
```
while `test` is better for:
```
test("the application should behave somehow")
```
In your tests, the latter option seems more suitable.